metricbeat/module/mongodb/collstats/_meta/fields.yml

```diff
@@ -0,0 +1,105 @@
+- name: collstats
+  type: group
+  description: >
+    MongoDB collection statistics metrics.
+  fields:
+    - name: db
+      type: keyword
+      description: >
+        Database name.
+
+    - name: collection
+      type: keyword
+      description: >
+        Collection name.
+
+    - name: name
+      type: keyword
+      description: >
+        Combination of database and collection name.
+
+    - name: statistics
+      type: group
+      description: >
+        Collection statistics as totals.
+      fields:
+        - name: total.time
+          type: long
+          description: >
+            Total waiting time for locks in microseconds.
+        - name: total.count
+          type: long
+          description: >
+            Total number of lock wait events.
+
+        - name: readLock.time
```

Comment:
To better follow our naming conventions, this should be `lock.read.time`, `lock.read.count`,  `lock.write.time`, ....

https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html